src/tests/fs_tests.rs

```diff
@@ -868,7 +869,8 @@ mod fs_tests {
         
         assert_eq!(cage.mkdir_syscall("/getdents", S_IRWXA), 0);
         let fd = cage.open_syscall("/getdents", O_RDWR, S_IRWXA);
-        assert_eq!(cage.getdents_syscall(fd, baseptr, bufsize), 48);
+        use std::convert::TryInto;
+        assert_eq!(cage.getdents_syscall(fd, baseptr, bufsize.try_into().unwrap()), 48);
```

Comment:
Rather than use try_into, use an explicit cast

Reply:
Fixed with the last update.